x-pack/test/fleet_api_integration/apis/epm/setup.ts

```diff
@@ -18,10 +18,17 @@ export default function (providerContext: FtrProviderContext) {
   const log = getService('log');
   const es = getService('es');
 
+  const uninstallPackage = async (name: string, version: string) => {
+    await supertest.delete(`/api/fleet/epm/packages/${name}/${version}`).set('kbn-xsrf', 'xxxx');
```

Comment:
thanks for fixing!
I checked locally and I think you have to add a force flag otherwise the package delete fails as there are package_policies created.

```
await supertest.delete(`/api/fleet/epm/packages/${name}/${version}`).set('kbn-xsrf', 'xxxx').send({force: "true"})
```

Reply:
Ah, thanks for the tip 👌